srcpkgs/linux5.5/files/x86_64-dotconfig

```diff
@@ -378,20 +378,19 @@ CONFIG_X86_MCE=y
 CONFIG_X86_MCE_INTEL=y
 CONFIG_X86_MCE_AMD=y
 CONFIG_X86_MCE_THRESHOLD=y
-CONFIG_X86_MCE_INJECT=m
+# CONFIG_X86_MCE_INJECT is not set
```

Comment:
Why?

Reply:
Is disabled since at least 4.19, could be enabled. I doubt anyone uses it.
"If you don't know what a machine check is and you don't do kernel QA it is safe to say n."